### PR TITLE
feat: adr 27 - port loading screen to Unity

### PR DESCRIPTION
**Pull request by AjimenezDCL**

### CODE REVIEW

File: docs/ADR-27-port-loading-screen-to-unity-.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Approach
+
+The _Loading Screen_ will be ported to _Unity_ as a _HUD_. _kernel_ will maintain the ownership of the visibility of the screen. Now, instead of changing the `showLoadingScreen` flag, _kernel_ will send a message to _Unity_ with the relevant information (`LoadingScreen: on|off, IsTeleporting: on|off, Stage: ESTABLISHING_COMMS | COMMS_ESTABLISHED | EXPERIENCE_STARTED...`)
+
+The _website_ _Loading Screen_ shouldn't be needed anymore. Currently we are dispatching some events listened by the _website_ about the renderer being loaded. Although _website_ is reacting properly to this, it's a never reached state since we don't show the player the `Play` or `Enter as guest` buttons until the renderer is ready.
+
+Therefore the whole _Loading Screen_ can be removed from _website_.
+
+The need of a preloading stage (while loading _Unity_) is out of this ADR scope. It would maintain the current implementation of a spinner hiding the `Play/Enter as guest` buttons.
```
Comment (AjimenezDCL) [inline docs/ADR-27-port-loading-screen-to-unity-.md:56]: @menduz Would you mind confirming I'm not mistaken here?

**Review by menduz — APPROVED** [review]
Looking good

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@
 - [ADR-24 - Decouple Kernel and Unity APIs](docs/ADR-24-decouple-kernel-and-unity-apis.md)
 - [ADR-25 - Explorer Repositories Decoupling](docs/ADR-25-explorer-repositories-decoupling-.md)
 - [ADR-26 - Port signup screen to Unity](docs/ADR-26-port-signup-screen-to-unity-.md)
+- [ADR-27 - Port signup screen to Unity](docs/ADR-27-port-loading-screen-to-unity-.md)
```
Comment (kuruk-mm) [inline README.md:29]:
```suggestion
- [ADR-27 - Port loading screen to Unity](docs/ADR-27-port-loading-screen-to-unity-.md)
```

**Review by kuruk-mm — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (AjimenezDCL) [reply README.md]: Oopsie!

**Pull request merged**